Assets/LeapMotionModules/InteractionEngine/Scripts/Editor/InteractionManagerEditor.cs

```diff
@@ -29,6 +29,9 @@ public class InteractionManagerEditor : CustomEditorBase<InteractionManager> {
                                 "_interactionNoClipLayer",
                                 "_brushLayer");
 
+      specifyConditionalDrawing(() => serializedObject.FindProperty("_graspingEnabled").boolValue,
```

Comment:
You can just write 
`specifyConditionalDrawing("_graspingEnabled", "_twoHandedGrasping") `
for this specific case